### PR TITLE
Fix performance dashboard build & Upgrade Django version

### PR DESCRIPTION
**Pull request by luukasmakila**
Fix performance dashboard build. Part of plans to improve the performance dashboard and benchmarking.

Summary of changes:
- Updated ubuntu image from bionic to jammy
- Cleaned up Makefile and modified the build command. Also added make run command.
- Updated pandas version in requirements.
- Updated Django to version 4.1.3 (Some of the old code was no longer supported so had to make minor changes)

Contributes to: https://github.com/istio/istio/issues/41609

### CODE REVIEW

File: perf_dashboard/Makefile [hunk 1]
```diff
@@ -15,16 +17,19 @@ get-cluster-credentials:
 	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
 
 build:
-	@docker build --no-cache --tag "$(IMG_LATEST)" --tag "$(IMG_VERSION)" .
+	@$(CONTAINER_CLI) build --http-proxy --tag "$(IMG_LATEST)" --tag "$(IMG_VERSION)" .
+
```
Comment (poussa) [inline perf_dashboard/Makefile:21]: Don't change the build args.

**Comment by ericvn** [comment]
/ok-to-test

Also need the CLA check satisfied @luukasmakila

**Comment by howardjohn** [comment]
Any chance we can update Django as well? The version we are on has CVEs

**Comment by poussa** [comment]
> Any chance we can update Django as well? The version we are on has CVEs

yeah, lets do that as well. @luukasmakila

**Comment by luukasmakila** [comment]
> 

I've been added to the correct google group now, not sure how long it will take to sync.

**Comment by luukasmakila** [comment]
> /ok-to-test
> 
> Also need the CLA check satisfied @luukasmakila

I've been added to the correct google group now, not sure how long it will take to sync.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**